LIPs/LIP-101.md

```diff
@@ -0,0 +1,46 @@
+---
+lip: 101
+title: Restart Treasury Reward Cut
+author: Ben Perez (@b3nnnp)
+type: Parameter
+status: Draft
+created: 2025-12-12
+discussions-to: https://forum.livepeer.org/t/pre-proposal-lip-101-restart-treasury-reward-cut
+requires: 91, 92
```

Comment:
I don't believe that we need this `requires` line, as those LIPs are already passed. I think requires should only be used if this LIP is dependent upon another LIP passing in conjunction.